DEPRECATIONS.md

```diff
@@ -4,6 +4,7 @@ This document lists deprecated constructs in the SDK, with their expected time 
 
 | API / Feature | Deprecated (warning) | Deprecated (error) | Removed | Notes |
 | --- | --- | --- | --- | --- |
+| `ChatDomain#replayEventsForActiveChannels` | 2021.11.10<br/>4.21.0 | 2021.11.10<br/>⌛ | 2021.11.10 ⌛ | Use `ChatClient#replayEventsForActiveChannels` instead |
```

Comment:
If you introduce deprecation you also should define periods when it gets deprecated with error and removed completely. Usually it 1 month, but can be two weeks. Read more [here](https://github.com/GetStream/InternalAndroidGuidelines#deprecations)

Reply:
Yes, but it has to be updated when we do a release. Isn't it?